### PR TITLE
Add quiet mode

### PR DESCRIPTION
**Pull request by kasium**

### CODE REVIEW

**Comment by kasium** [comment]
Resolves #160

File: diff_cover/tests/test_integration.py [hunk 1]
```diff
@@ -393,6 +393,13 @@ def test_git_diff_error(self):
         with self.assertRaises(CommandError):
             diff_cover_main(["diff-cover", "coverage.xml"])
 
+    def test_quite_mode(self):
```
Comment (Bachmann1234) [inline diff_cover/tests/test_integration.py:396]: This and the other test both say `quite` rather than `quiet`

**Comment by Bachmann1234** [comment]
Rough inspection looks good. Update this from master so CI runs against it. I wanna look that over before merging/releasing this.

Thanks for all the work btw!

**Comment by kasium** [comment]
@Bachmann1234 done

**Comment by Bachmann1234** [comment]
@kasium build looks good. If you fix my nitpick comment https://github.com/Bachmann1234/diff_cover/pull/197/files#r651310535 ill merge and release this

File: diff_cover/tests/test_integration.py (hunk 1 above)
Comment (kasium) [reply diff_cover/tests/test_integration.py]: done

**Comment by kasium** [comment]
@Bachmann1234 done

**Comment by Bachmann1234** [comment]
@kasium awesome, ill merge this now. I wanna get a release out today but it might not be til tomorrow.

**Pull request merged**